core/soc/estimator.go

```diff
@@ -71,9 +83,7 @@ func (s *Estimator) RemainingChargeDuration(chargePower float64, targetSoC int)
 			}
 		}
 
-		// estimate remaining time
-		whRemaining := percentRemaining / 100 * s.virtualCapacity
-		return time.Duration(float64(time.Hour) * whRemaining / chargePower).Round(time.Second)
+		return s.AssumedChargeDuration(targetSoC, chargePower)
```

Comment:
Warum müssen wir die Funktion hier ändern? Du hast oben ja bereits eine neue eingeführt. Braucht es beide? Sind die Vehicle APIs jetzt überhaupt noch relevant oder sollen sie entfernt werden?

Reply:
In der obigen Funktion wurde der gemeinsame Nenner rausgezogen. Um möglichst wenig Code zu duplizieren ruf ich einfach die neue Funktion auf.